docs/docsearch-scraper-config.json

```diff
@@ -0,0 +1,63 @@
+{
+  "index_name": "dev_docs",
+  "start_urls": [
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
+    },
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/.+$",
+      "selectors_key": "folder",
+      "page_rank": 0,
+      "tags": [
+        "folder"
+      ]
+    },
+    {
+      "url": "https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.+\\.md$",
+      "selectors_key": "file",
+      "page_rank": 5,
+      "tags": [
+        "file"
+      ]
+    }
+  ],
+  "selectors": {
+    "folder": {
+      "text": ".js-navigation-item td.content"
+    },
```

Comment:
For the folder view pages, this tells the scraper to look only at the links in this table:
![image](https://user-images.githubusercontent.com/12970166/74692293-6dac2380-519b-11ea-85f8-9b115f664643.png)